app/views/wikies/index.html.erb

```diff
@@ -29,7 +29,7 @@
 <br>
 
 <%= link_to 'New Wiky', new_wiky_path %>
-
+<% if (current_user.premium? || current_user.admin?) %>
```

Comment:
No need to duplicate this expression that's also in `WikiesController#index`. Instead just rely on `@private`:

```ruby
<% if @private.any? %>
```